File: examples/emoji/flag/go/getFlagByCountryCode.go
package example

import (
	"fmt"
	"github.com/micro/services/clients/go/emoji"
	"os"
)

// Get the flag for a country. Requires country code e.g GB for great britain
func GetFlagByCountryCode() {
	emojiService := emoji.NewEmojiService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := emojiService.Flag(&emoji.FlagRequest{})
	fmt.Println(rsp, err)
}

File: examples/js/emoji/flag/getFlagByCountryCode.js
const { EmojiService } = require("m3o/emoji");

// Get the flag for a country. Requires country code e.g GB for great britain
async function getFlagByCountryCode() {
  let emojiService = new EmojiService(process.env.MICRO_API_TOKEN);
  let rsp = await emojiService.flag({
    alias: "GB",
  });
  console.log(rsp);
}

getFlagByCountryCode();
